cmd/log-parser/README.md

```diff
@@ -0,0 +1,94 @@
+# `kata-log-parser`
+
+* [Logfile requirements](#logfile-requirements)
+* [Component logfiles](#component-logfiles)
+* [Usage](#usage)
+
+`kata-log-parser` is a tool that combines logfiles generated by the various
+system components, sorts them by timestamp and re-displays the log entries,
+adding a time delta showing how much time has elapsed between each log entry.
+
+The tool also checks the validity of all log records and can re-format the
```

Comment:
Lines 11-12 suggested rewrite: The tool also checks the validity of all log records, can re-format the logs, and output them in a different format.